packages/template-retail-react-app/app/components/product-view-modal/bundle.test.js

```diff
@@ -166,6 +166,20 @@ test('bundle product view modal disables update button when quantity exceeds chi
         expect(swingTankProductView).toBeInTheDocument()
         expect(sizeSelectBtn).toBeInTheDocument()
         expect(quantityInput).toBeInTheDocument()
+    })
+
+    // Wait for all child products to be rendered
+    await waitFor(() => {
+        expect(screen.getAllByText('Sleeveless Pleated Floral Front Blouse')).toHaveLength(2)
+        expect(screen.getAllByText('Swing Tank')).toHaveLength(2)
+        expect(screen.getAllByText('Pull On Neutral Pant')).toHaveLength(2)
+    })
```

Comment:
is any of these products a standard/simple product? 
It's not clear to me whether we have new test case to exercise the inclusion of a simple product in a bundle.